container/Dockerfile

```diff
@@ -7,25 +7,26 @@
 #   verify about certificates
 #   correct openssl_config URL location
 #   lcgBundle now fix version
-#   start mySQL
-#   install/start ElasticSearch (?)
-#   install/start ActiveMQ (?)
+#   start mySQL + install/start ElasticSearch (?) + install/start ActiveMQ (?) or better use docker-compose
+#
 
 
 # CC7 as base (the latest tag doesn't work!)
 FROM cern/cc7-base:20170113
 MAINTAINER Federico Stagni <federico.stagni@cern.ch>
 
 # Create DIRAC dirs
-RUN mkdir -p /opt/dirac/DIRAC
-RUN mkdir -p /opt/dirac/etc/grid-security/certificates
+RUN mkdir -p /opt/dirac/DIRAC && \
+    mkdir -p /opt/dirac/etc/grid-security/certificates && \
+    mkdir -p /opt/dirac/user && \
+    cd /opt/dirac
 
 # Installing DIRAC in /opt/dirac
-WORKDIR /opt/dirac
 RUN curl -L -o dirac-install https://raw.githubusercontent.com/DIRACGrid/DIRAC/integration/Core/scripts/dirac-install.py && \
     chmod +x dirac-install && \
-    ./dirac-install -r v6r17 -t server -i 27 -g 2017-01-27
-RUN rm -rf /opt/dirac/.installCache && rm dirac-install
+    ./dirac-install -r v6r17 -t server -i 27 -g 2017-01-27 && \
```

Comment:
Ah right...
It is a good idea to put the lines that will potentially change at the end because when rebuilding an image, it starts from the first line that changes. The previous layers are taken from the cache. So by putting at the end what changes, you gain time and save space